docs/source/release_notes.rst

```diff
@@ -1,13 +1,22 @@
 .. ...........................................................................
-.. © Copyright IBM Corporation 2020, 2021, 2023                              .
+.. © Copyright IBM Corporation 2020 - 2024                              .
```

Comment:
seems like we've been preferring to use commas for copyright: `2019, 2024`

Reply:
done